packages/sources/coinmetrics-test/src/endpoint/price-router.ts

```diff
@@ -31,6 +31,7 @@ export const routingTransport = new RoutingTransport<AssetMetricsEndpointTypes>(
 
 export const endpoint = new AdapterEndpoint<AssetMetricsEndpointTypes>({
   name: 'price-ws',
+  aliases: ['crypto'],
```

Comment:
You should be able to use the CryptoPriceEndpoint class here instead (and PriceAdapter for the Adapter) that will give you these aliases 😄

Reply:
Done 👍